packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -1514,7 +1512,7 @@ class _TransitionableNavigationBar extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     assert(() {
-      bool inHero;
+      bool? inHero;
```

Comment:
Can't this be non-nullable and default to false?

Reply:
Done